tutorials/abap-environment-create-rap-business-events/abap-environment-create-rap-business-events.md

```diff
@@ -0,0 +1,543 @@
+---
+title: How to Create RAP Business Events in SAP BTP ABAP Environment
+description: Create and exposure of business events in SAP BTP ABAP Environment
+auto_validation: true
+time: 45
+tags: [ tutorial>beginner, programming-tool>abap-development, software-product>sap-business-technology-platform, tutorial>license]
+primary_tag: software-product>sap-btp--abap-environment
+author_name: Niloofar Flothkoetter
+author_profile: https://github.com/niloofar-flothkoetter
+---
+
+## Prerequisites
+ - You need to have access to a SAP BTP ABAP environment system.
+ - You need to prepare an event mesh instance in your SAP business technology platform system and download the service key of this instance. For more information see the [Create Instance of SAP Event Mesh] (https://developers.sap.com/tutorials/cp-enterprisemessaging-instance-create.html)
+ - You need a user with access to maintain communication arrangement.
+
+## Details
+### You will learn
+  - How to create RAP Events Business
+  - How to set up a channel to connect to SAP Event Mesh
+
+>Always replace `####` with your initials or group number.
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Overview)]
+
+The ABAP RESTful Application Programming Model (RAP) now supports the native consumption and exposure of business events. For exposure, an event can be defined and raised in a RAP business object or in the behavior extension and then published via Event Bindings.
```

Comment:
Typically, a step would include instructions to guide a user through doing something that ends with a result, an accomplished task.  The test you have in this step actually belongs in the intro section; you can add it after row 22.  You can then move all the other steps up.